test_files/tycheck/type_error_msgs/invalid_arg.erl

```diff
@@ -1,5 +1,5 @@
 % ERROR
-% test_files/tycheck/type_error_msgs/invalid_arg.erl:13:1: Type error: function make_even/1 failed to type check against type fun((integer()) -> integer())
+% test_files/tycheck/type_error_msgs/invalid_arg.erl:13:17: Type error: expression failed to type check
```

Comment:
Do we lose information for the new error messages?

Reply:
No. In fact, several error message became more precise.